packages/@n8n_io/eslint-config/base.js

```diff
@@ -412,6 +417,20 @@ const config = (module.exports = {
 				'@typescript-eslint/no-unused-vars': 'off',
 			},
 		},
+		// @TODO Remove these exceptions
+		{
+			files: [
+				'./packages/cli/**/*.ts',
+				'./packages/workflow/**/*.ts',
+				'./packages/core/**/*.ts',
+				'./packages/node-dev/**/*.ts',
+				'./packages/editor-ui/**/*.ts',
```

Comment:
since this is missing `.vue` files, and cypress tests, I think it'd be better to revert the changes to this file, and add the rule only in `packages/nodes-base/.eslintrc.js`